Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -215,6 +218,10 @@ class BrowserViewController: UIViewController {
         }
     }
 
+    func SELappDidEnterBackgroundNotification() {
+        displayedPopoverController?.dismissViewControllerAnimated(true, completion: nil)
```

Comment:
Since we're in the background and presumably not visible, perhaps we should use `false` here?